README.md

```diff
@@ -40,6 +41,10 @@ This repository contains the source code of the key components that works togeth
 
     Click [here →](./examples/multi-brand-identity/) for the source code 🧑‍💻
 
+## Documentation
+
+For more information on how to use Oxygen UI, check out the documentation website.
```

Comment:
Shall we add the link to the site here?

Reply:
Fixed with https://github.com/wso2/oxygen-ui/pull/71/commits/97d38a2171ddcbbec9b77e2d2976ecad9c109698. Thanks 🙌.